modules/lib/src/main/resources/assets/js/app/ContentVersions.ts

```diff
@@ -4,39 +4,31 @@ import {GetContentVersionsForViewResultsJson} from './resource/json/GetContentVe
 
 export class ContentVersions {
 
-    readonly contentVersions: ContentVersion[];
+    private readonly contentVersions: ContentVersion[];
 
-    constructor(contentVersions: ContentVersion[]) {
+    private readonly activeVersionId?: string;
+
+    constructor(contentVersions: ContentVersion[], activeVersionId?: string) {
         this.contentVersions = contentVersions;
+        this.activeVersionId = activeVersionId;
     }
 
     getContentVersions(): ContentVersion[] {
```

Comment:
Let's rename this to `get`. Now we have `this.contentVersions.getContentVersions()` all over the place and it will be unambiguous (and look better) as `this.contentVersions.get()`